src/main/java/picard/fingerprint/CrosscheckFingerprints.java

```diff
@@ -253,6 +263,16 @@ public class CrosscheckFingerprints extends CommandLineProgram {
             doc = "One or more input files (or lists of files) with which to compare fingerprints.", minElements = 1)
     public List<String> INPUT;
 
+    @Argument(doc = "A tsv with two columns and no header which maps the input files to corresponding indices; to be used when index " +
+            "files are not located next to input files. First column must match the list of inputs. ", optional = true)
```

Comment:
This strategy was originally thought through, but there were a few issues that came up that led me to abandon that path for this particular tool (after chatting with David R & Chris K about it too). Although it would be nice to match David's recent GATK PR paradigm, here are some problems with trying to implement it for this particular tool:

* There are actually three different ways to input sample information into CrosscheckFingerprints. There's the `INPUT` argument, the `INPUT_SAMPLE_MAP`, and the `INPUT_SAMPLE_FILE_MAP`. (And corresponding ones for the `SECOND_INPUT`…) These have different use cases for different parts of the downstream analysis.
* For this particular patch, we need to intervene at the moment when a VCF / SAM file is actually parsed by the tool to ensure index files can be provided. After inspecting the tool, I saw that only the `INPUT` arguments were ever fed to a parser. Hence, one can only intervene at the level of those file paths getting resolved. In fact, those are the only arguments which include paths at all.
* This meant it became necessary to allow the user to specify where the index files were for the `INPUT` paths specifically. The problem with trying to absorb this into the `INPUT_SAMPLE_MAP` or other similarly named file is that those don't actually contain a column with `INPUT` paths (hence why those get fed into `getStringStringMap` rather than a Path map). So we needed a totally new type of table which matched the file paths with index paths.

I agree longterm it'd be nice to adapt the UI a bit to agree with the other solution in the GATK tool, but it seems it would be a bit of a heavier lift than this was to implement.

Reply:
That's reasonable, thank you for explaining.  If we're going to be using these sample maps every where it would be nice of us to promote them to a more official thing and stick a sane parser in htsjdk or picard so we don't have to reinvent them repeatedly, but not the job of this pr.